office-for-students/wagtail-CMS
# Feature/ofs 172 add link to covid banner

### [OFS-172](https://mobilisecloud.atlassian.net/browse/OFS-172)

Added link to Downloadable Resources page to Covid-19 bar (english version only)

### How to review
- Point Wagtail to Postgres DB in Pre-prod
- Go to http://localhost:8000/corona-covid-19-information/
- Check that you see "For more information and advice, please refer to our downloadable resources for students, teachers and advisers." text at the bottom of the COVID-19 banner
- Check that the text is not displayed in Welsh version of the page
- Click on "downloadable resources" link, you should be directed to Downloadable Resources page: http://localhost:8000/how-do-i-choose-course/downloadable-resources/
- Check this works the same on Course Details and Compare Courses pages
